docs/creating_review_app.md

```diff
@@ -38,70 +17,65 @@ CircleCI will match the `review-app-` prefix and either:
 2. Update an existing review app using `update_review_app.sh`
 3. Once CI is complete follow [this step](https://github.com/artsy/force/blob/master/docs/creating_review_app.md#dns-setup) to setup DNS.
 
-> Once you're done working with the review app it (currently) has to be deleted manually
-
-#### Deleting a Review App
-
-Run:
-
-```sh
-yarn delete-review-app <name>
-```
-
-The `name` is the name of the branch, minus the `review-app-` prefix.
-
-#### Manual Steps
-
-##### Manual Build
+### Building on Local
 
 First, make sure `jq` is installed:
 
 ```sh
 brew install jq
 ```
 
-Then launch the script:
+Then launch the script with:
 
 ```sh
-./scripts/build_review_app.sh review-app-name
+./scripts/build_review_app.sh awesome-feature
 ```
 
-##### Manual Update
+The script will save a K8s spec in `hokusai/awesome-feature.yml`
 
-If you want to push subsequent changes to the review app you can push a new build to the same tag with the `--overwrite` flag:
+### Accessing the review app.
 
-```sh
-hokusai registry push --overwrite --skip-latest --force --tag <name>
-```
+To access the review app, you must create a DNS name for it. A name ending in `artsy.net` is required for full OAuth flows (for, say, login redirects between Gravity and Force for Auction registration). On Cloudflare, please do:
 
-and you need to redeploy your app:
+1. [Login to Cloudflare](https://dash.cloudflare.com/), and navigate to **artsy.net** > **DNS**
+1. Click `+ Add Record`
+1. Change `Type` dropdown to `CNAME`
+1. Under `Name` enter `awesome-feature`
+1. Under `Target` say `nginx-staging.artsy.net`
+1. Hit `Save`
+1. DNS will propagate and after a few minutes the review app will be available via `awesome-feature.artsy.net`
+
+### Updating the review app
+
+If you want to push subsequent changes to the review app, push a new build to the same tag with the `--overwrite` flag:
```

Comment:
Worth noting here that review apps created via pull requests will update on pushes to feature branch.

Reply:
Good catch. Done.